packages/expo-dev-menu/ios/DevMenuManager.swift

```diff
@@ -387,11 +369,11 @@ open class DevMenuManager: NSObject, DevMenuManagerProtocol {
       return false
     }
     if visible {
-      guard let bridge = delegate?.appBridge?(forDevMenuManager: self) as? RCTBridge else {
-        debugPrint("DevMenuManager: The delegate is unset or it didn't provide a bridge to render for.")
+      guard let bridge = currentBridge else {
+        debugPrint("DevMenuManager: There is no bridge to render DevMenu.")
         return false
       }
-      session = DevMenuSession(bridge: bridge, appInfo: delegate?.appInfo?(forDevMenuManager: self), screen: screen)
+      session = DevMenuSession(bridge: bridge, appInfo: nil, screen: screen)
```

Comment:
is it correct to pass `nil` here, rather than including the build/app info from the manifest?

Reply:
nice catch! thanks

Reply:
after taking a second look at this - `DevMenuSession` is actually no longer needed - I've updated the PR to remove it